src/CommunityToolkit.Maui.Core/Handlers/SemanticOrderView/SemanticOrderViewHandler.tizen.cs

```diff
@@ -3,10 +3,10 @@
 
 namespace CommunityToolkit.Maui.Core.Handlers;
 
-public partial class SemanticOrderViewHandler : Microsoft.Maui.Handlers.ElementHandler<ISemanticOrderView, object>
+public partial class SemanticOrderViewHandler
 {
 	/// <inheritdoc/>
-	protected override object CreatePlatformElement()
+	protected override ContentViewGroup CreatePlatformElement()
```

Comment:
Should be `CreatePlatformView()`? with `using Microsoft.Maui.Platform;`.

Reply:
Later I will check the Tizen implementation for `MauiSemanticOrderView`!

Reply:
Thank you @JoonghyunCho !